pkg/controller/deadmanssnitchintegration/deadmanssnitchintegration_controller.go

```diff
@@ -637,3 +640,8 @@ func getCondition(conditions []hivev1.ClusterDeploymentCondition, t hivev1.Clust
 	}
 	return nil
 }
+
+func getClusterID(clusterNS string) string {
```

Comment:
this can probably be removed since it's only used in one place

Reply:
@sam-nguyen7 I would keep this function if you pull in functionally to actually grab the data from the API as well. Having just one line of code wrapped in a function is a bit of over engineering.

Reply:
@sam-nguyen7 I think you also be able to get the clusterID from a label in the Clusterdeployment. That could live in here.

Reply:
i like having it in a function. more space to grow. let us say that we want to get/parse the ClusterID in the future from somewhere else or another file. Then we already have a function that does it. I would put it on a utils package though.